reStream.sh

```diff
@@ -28,6 +29,27 @@ while [ $# -gt 0 ]; do
             shift
             shift
             ;;
+        -w | --webcam)
+            webcam=true
+            format="v4l2"
+
+            # check if there is a modprobed v4l2 loopback device
+            # use the first cam as default if there is no output_path already
+            cam_path=$(v4l2-ctl --list-devices \
+                | sed -n '/^[^\s]\+platform:v4l2loopback/{n;s/\s*//g;p;q}')
+
+            # fail if there is no such device
+            if [ -e "$cam_path" ]; then
+                if [ "$output_path" = "-" ]; then
+                    output_path="$cam_path"
```

Comment:
This will possibly give problems when you first specify an alternative webcam path with `-o` and then use the `-w` toggle.

Maybe move the initialization to the block at line 125? That's also a bit cleaner I think.

Reply:
I don't think this will give a problem because I use `$cam_path` (not `$output_path`) to check if the device exists v4l2 is loaded.

If there already was a `-o` I do not overwrite it.

Reply:
Ah hmm. You're right. I still find it odd the webcam initialization is separate though. But I'll probably refactor the ffmpeg stuff anyway later.